### DISCUSSION TITLE
Where can I download the scss folder for RTL?

### DISCUSSION THREAD

**Discussion by sonvnn** [twbs/bootstrap#40819] [Q&A]
Where can I download the scss folder for rtl? Sorry I can't find it. In the source code there is only the scss file for LTR.

Thanks & Best Regards,
Sonny

**Comment by julien-deramond** [comment]
Hello @sonvnn 

There's no specific folder for RTL. We use the same SCSS files with `/* rtl:* */` "commands" in them; see https://github.com/twbs/bootstrap/blob/fecd219983539aa0110a7e09d31b8aa4d5fb4348/scss/_popover.scss#L78-L102

The generated CSS file is, however, different, and available at https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.rtl.css
